### ISSUE TITLE
Radial club hand

### ISSUE THREAD

**Issue by evanc1648** [obophenotype/human-phenotype-ontology#11101]
**HPO term**
Name: Radial club hand
ID: HP:0004059
Link: https://hpo.jax.org/browse/term/HP:0004059

**Synonym to be added**
Type IV radial deficiency

Source: PMID:21483577

**broad/exact/narrow/plural/abbreviation ?**
exact

**Comment by pnrobinson** [comment]
@evanc1648 thanks for pointing this out

Radial ray deficiency is a spectrum of congenital abnormalities affecting the preaxial (radial) side of the upper limb.  According to https://pmc.ncbi.nlm.nih.gov/articles/PMC5447903/

Bayne et al. classified radial ray deficiency into four types based on the radiographic severity of the radial ray deficiency: Type I, short distal radius (radius is slightly shorter than ulna, distal growth plate is present); Type II, hypoplastic radius (radius is smaller/thinner than ulna, no growth plate is present); Type III, partial absence of the radius (only a small proximal radial segment is present); and Type IV, total absence (radius is completely absent, ulna may be curved)

Radial ray deficiency is thus "bundled". Radial club hand can be a feature of Type III and IV.

@azankl -- I am wondering how to model this.
Currently we have a term [Abnormal radial ray morphology HP:0410049](https://hpo.jax.org/browse/term/HP:0410049) that probaby should be the parent of this term? Can we say that Radial club hand is a synonym of radial ray deficiency though, or is it a slightly different concept?

**Comment by azankl** [comment]
@pnrobinson According to your reference, radial club hand 'has a wide range of phenotypes from hypoplasia of the thumb to complete absence of the radius and the first ray.' So I would say, radial club hand and radial ray deficiency are synonymous (I would prefer radial ray deficiency as the main label) and abnormal radial ray morphology should be the parent term, as you suggested.

**Comment by pnrobinson** [comment]
@azankl  @evanc1648 
OK, so I will revise the term [Abnormal radial ray morphology HP:0410049
](https://hpo.jax.org/browse/term/HP:0410049)
as follows. Let me know if anything is missing or wrong!

Label: Radial ray deficiency
synonyms:Abnormal radial ray morphology, Radial club hand
definition: Radial club hand or radial ray deficiency is a complex congenital abnormality of the radial or pre-axial border of the upper extremity. The term radial club hand is widely used and accepted to describe congenital longitudinal radial ray deficiency, that is, a hand that is radially deviated at the distal forearm in the shape of a golf club.
PMID:28603568
PMID:3559066

Comment: Bayne et al. classified radial ray deficiency into four types based on the radiographic severity of the radial ray deficiency: Type I, short distal radius (radius is slightly shorter than ulna, distal growth plate is present); Type II, hypoplastic radius (radius is smaller/thinner than ulna, no growth plate is present); Type III, partial absence of the radius (only a small proximal radial segment is present); and Type IV, total absence (radius is completely absent, ulna may be curved)

**Comment by pnrobinson** [comment]
done
